src/router/index.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable radix */
+
+'use strict';
+
+const express = require('express');
+const db = require('./../db');
+const products = require('./../db/products.json');
+
+const router = express.Router();
+
+// router.get('/', productsRoutes.allProducts);
+
+// creating a route - route gets all product
+router.get('/', (req, res) => res.json(products));
```

Comment:
I thought here we are supposed to display a bootstrap page with a table layout of list of products rather than json output.